packages/design-system/src/components/__snapshots__/i18n.test.ts.snap

```diff
@@ -1,29 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`i18n getTranslations and addTranslations getTranslation returns core English translations 1`] = `"Official government website"`;
+exports[`i18n getTranslations and addTranslations getTranslation returns core English translations 1`] = `"Official website of the United States government"`;
```

Comment:
Are these all updates from the latest version of USWDS or something?

Reply:
Ah yes, USWDS had one change with the English `aria-label` that I updated.

Reply:
Oh - and we were using the wrong apostrophes in our JSON file. I think it was straight vs curly apostrophes? Not sure, but it was flagged as an incorrect symbol in VS Code